utils/RequestUtil.js

```diff
@@ -73,20 +73,23 @@ var RequestUtil = {
   json: function (options) {
     // Default assign options to empty object
     options || (options = {});
+    var requestID = options.url;
```

Comment:
So does this mean that each time the query params are different, that should result in a unique ID? Or should query params not affect the ID?